packages/assets/scss/01-atoms/_table.scss

```diff
@@ -5,6 +5,26 @@
   border-collapse: collapse;
   width: 100%;
 
+  .ma__table__caption,
+  & td > table caption {
+    color: $c-font-heading;
+    font-weight: $fonts-bold;
+    line-height: 1.3;
+    text-align: left;
+    margin-bottom: 0.8em;
+
+    &.hide {

```

Comment:
This seems to be reserve of what's being set for the scroll info above, see .ma__table__caption__scroll-info
Just want to make sure this is needed

Reply:
This hides the scroll info when the table has no scrollbar.  JS adds the class at evaluating the table width and its column width.

Reply:
ah I see, so the scroll info is controlled by the .show class and the title and scroll info are both controlled by .hide

Thanks for clarifying